homeassistant/components/sensor/miflora.py

```diff
@@ -0,0 +1,142 @@
+"""
+Support for Xiaomi Mi Flora BLE plant sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.miflora/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.const import CONF_MONITORED_CONDITIONS, CONF_NAME
+
+
+REQUIREMENTS = ['miflora==0.1.4']
+
+LOGGER = logging.getLogger(__name__)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=900)
+CONF_MAC = "mac"
+CONF_FORCE_UPDATE = 'force_update'
+DEFAULT_NAME = ""
```

Comment:
So, `DEFAULT_NAME` would not be needed at all and instead doing the test for an empty string it could be done against `None`.